packages/utils/core-utils/src/package-manager.ts

```diff
@@ -0,0 +1,44 @@
+import { CrossSpawnArgs, CrossSpawnOptions, spawn } from '@malept/cross-spawn-promise';
+import chalk from 'chalk';
+import logSymbols from 'log-symbols';
+import yarnOrNpm from 'yarn-or-npm';
```

Comment:
You have removed this library in `packages/api/core/package.json` and should not use it anymore.

https://github.com/electron/forge/pull/3351/files#diff-05551007aaa07544d8011259e77a3e3d973269bf2707d5cdf2180bdd50801e5bL76